packages/adders/lucia/index.ts

```diff
@@ -365,7 +392,7 @@ export default defineAdder({
 				const [ts] = utils.createPrinter(typescript);
 				return dedent`
 					import { hash, verify } from '@node-rs/argon2';
-					import { generateRandomString } from '@oslojs/crypto/random';
+					import { encodeBase32LowerCaseNoPadding } from '@oslojs/encoding';
```

Comment:
thanks! are there some docs where I can learn more about this method and others that are available? I didn't see it mentioned on https://oslo.js.org/reference/encoding/

Reply:
Sorry, that's the "v1" docs. The "v2" successor is available at https://encoding.oslojs.dev and it's documented here

https://encoding.oslojs.dev/reference/main/encodeBase32LowerCaseNoPadding

Reply:
Base32 is much more compact than hex encoding (used by UUID etc) but it's also case insensitive and doesn't use special characters unlike base64

Reply:
I would lean towards the base 64 version to maintain the previous behavior as it's a much more compact representation and more closely matches popular implementations like https://github.com/ai/nanoid

Reply:
I personally prefer IDs that don't contain special characters and this provides about the same entropy as nanoid with a similar number of characters (24 vs 21). I don't mind changing it though. Do you want me to change `generateSessionToken()` to use base64 as well?

Reply:
Yeah, I guess it makes sense to remain consistent

I see the input was changed to 15 characters. How do we know how many characters the output will be?

Reply:
The generated ID will actually be 24 characters. In base32, every 5 bits (that's where the 32 comes from) are converted to a corresponding character (e.g. 0b00001 => A). So you get 8 characters for every 5 bytes, or 24 characters for a 15 byte input.

For base64, it's 6 bits instead of 5, so you get 4 characters for every 3 bytes.